mcodingbot/plugins/highlights.py

```diff
@@ -219,7 +215,7 @@ async def on_message(event: hikari.GuildMessageCreateEvent) -> None:
                 highlights[user_id].append(highlight)
 
     for user_id, hls in highlights.items():
-        if sent_message_cooldown.get_retry_after(
+        if sent_message_cooldown.trigger(
```

Comment:
This should be .retry_after

Reply:
This should be .retry_after, or `if not sent_message_cooldown.can_trigger`. It shouldn't actually be triggering anything.